core-web/apps/dotcms-ui/src/app/view/components/_common/dot-workflows-actions-selector-field/dot-workflows-actions-selector-field.component.ts

```diff
@@ -56,8 +56,8 @@ export class DotWorkflowsActionsSelectorFieldComponent
             tap((actions: SelectItemGroup[]) => {
                 const actionsIds = this.getActionsIds(actions);
 
-                if (actionsIds.length && !actionsIds.includes(this.value)) {
-                    this.dropdown.clear(null);
+                if (this.dropdown && actionsIds.length && !actionsIds.includes(this.value)) {
```

Comment:
Approved, but, we can create a private method with a good name to understand why means the if (something like this: `hasActions(x,y)`)